src/chttpd_db.erl

```diff
@@ -61,7 +61,7 @@ handle_request(#httpd{path_parts=[DbName|RestParts],method=Method,
         do_db_req(Req, Handler)
     end.
 
-handle_changes_req(#httpd{method='GET'}=Req, Db) ->
+handle_changes_req1(#httpd{}=Req, Db) ->
```

Comment:
`handle_changes_req1` should comes after `handle_changes_req` functions. You should be able to read the code from top to bottom, not in reverse.